static/css/com/layout/footer.scss

```diff
@@ -3,103 +3,231 @@
 @import '../_icon/_icon';
 
 .global-footer {
-  height: $footer-height;
-  color: #b2b2b2;
-
-  .header {
-    font-family: $font-family-serif;
-    font-size: 24px;
-    color: #7c7979;
-    margin-bottom: $unit*3;
-  }
-
-  a {
-    color: #fff;
-    text-decoration: none;
-
-    &:hover {
-      text-decoration: underline;
-    }
-  }
+  color: rgba(39, 40, 44, 0.7);
+  width: 100%;
 }
 
 .global-footer-terms {
-  @include box-sizing;
   @include clearfix;
-  padding: $unit*3 0;
-  background-color: #27282C;
-  color: rgba(255, 255, 255, 0.5);
-  font-family: Inter, Arial, sans-serif;
+  padding: 32px 22px;
+  background-color: #F4F4F4;
+  min-width: 320px;
+  color: rgba(39, 40, 44, 0.7);
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Droid Sans, Helvetica Neue, Arial, sans-serif;
   font-size: 14px;
+  -webkit-font-smoothing: antialiased;
   line-height: 20px;
+  border-top: 1px solid rgba(39, 40, 44, 0.2);
+
+  @media (max-width: 425px) {
+    padding: 32px 16px
+  }
+
+  .g-footer-layout {
+    min-width: 960px;
+    max-width: 960px;
+    width: 100%;
+    margin: auto;
+    display: flex;
+    justify-content: space-between;
+
+    @media (max-width: 960px) {
+      width: 100%;
```

Comment:
exists in base selector for this rule and somethin else, does it need?

Reply:
It's redundant, I've deleted it. Thanks!